site/app/libraries/database/PostgresqlDatabaseQueries.php

```diff
@@ -929,6 +931,26 @@ public function getTeamsById(array $team_ids) {
         return $teams;
     }
 
+    /**
+     * Converts the result of array_to_string(*, ',') into a php
+     *  array using the provided conversion function
+     * @param string|mixed $str string array of values, or single value from database
+     * @param callback $parser A method to parse a string representation of the object (i.e. 'intval')
+     * @return array A php-array representation of the provided array string
+     */
+    private static function parseShallowCommaSeparatedArrayString($str, $parser) {
```

Comment:
Since we have Postgres > 9.2, we may want to use this instead: https://stackoverflow.com/a/13670706/214063